app/src/main/java/com/example/vybe/Vibe.java

```diff
@@ -0,0 +1,6 @@
+package com.example.vybe;
+
+import java.io.Serializable;
+
+public class Vibe implements Serializable {
+}
```

Comment:
Fill out fields and methods (getters and setters) so it is clear what needs to be changed in the UML